ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -92,6 +103,10 @@
     try {
       final Map<String, DcZkInfo> dataCenterToZkAddress =
           parseDcJsonAndPopulateDcInfo(clusterMapConfig.clusterMapDcsZkConnectStrings);
+      // Make sure the HelixManager of local datacenter gets connected first and partitionOverrideInfoMap use PropertyStore
+      // in local DC for initialization.
+      HelixManager localManager =
+          initializeHelixManagerAndPropertyStoreInLocalDC(dataCenterToZkAddress, instanceName, helixFactory);
```

Comment:
The intention here is to make sure clusterManager connects to HelixManager in local datacenter first. (There are two benefits: 1. the `PropertyStore` might vary among colos later, clusterManager should use `PropertyStore` in local DC.  2. There is race condition if clusterManager concurrently connects to 4 HelixManager in all DCs)

Reply:
The latest code changes and deployment plan are updated in Design doc. Also the race condition is discussed there.